src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -257,9 +258,9 @@ private function autowireMethod(\ReflectionMethod $reflectionMethod, array $argu
                 continue;
             }
 
-            $typeName = InheritanceProxyHelper::getTypeHint($reflectionMethod, $parameter, true);
+            $type = InheritanceProxyHelper::getTypeHint($reflectionMethod, $parameter, true);
```

Comment:
As all these renamings are already merged as part of another PR now, can you rebase your branch to make the diff more readable ?